src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/FocusManagerTests/UnoSamples_Tests.FocusManager.cs

```diff
@@ -10,15 +11,10 @@
 
 namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.FocusManagerTests
 {
+	[ActivePlatforms(Platform.iOS, Platform.Browser)]	// Ignore focus tests for android, focus is getting stolen incorrectly 
```

Comment:
Issue # for this?

Reply:
This one: https://github.com/unoplatform/uno/issues/1244